operator/roles/forkliftcontroller/tasks/main.yml

```diff
@@ -80,6 +80,11 @@
       definition: "{{ lookup('template', 'controller/provider-host.yml.j2') }}"
     when: "'kubevirt.io' in api_groups"
 
+  - name: "Setup populator deployment"
+    k8s:
+      state: present
+      definition: "{{ lookup('template', 'populator/deployment-populator.yml.j2') }}"
```

Comment:
would it make sense to check if the CRD is defined first to cope with a situation in which CDI deploys it first (assuming they'll use the same CRD - that's something we'll need to figure out)?